src/bandersnatch/main.py

```diff
@@ -15,11 +15,16 @@
 import bandersnatch.verify
 
 from .configuration import BandersnatchConfig
+from .filter import filter_project_plugins, filter_release_plugins
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)  # pylint: disable=C0103
 
 
 def mirror(config):
+    # Load the filter plugins so the loading doesn't happen in the fast path
```

Comment:
What is the fast path? Do you mean, lets load them here so we hit the fast path during bandersnatch runs?

Reply:
Yes, the load is slow so we want them loaded before the mirroring operation starts.  

Without this change, each worker will run the plugin load.  Which slows things down a bit.  But also runs the plugin initialization.

In the case of this plugin since it fetches safety_db from github during it's initialization it also results in it running the db download from github multiple times (since each worker load the plugins)